docs/source-pytorch/advanced/model_parallel/deepspeed.rst

```diff
@@ -408,6 +408,7 @@ Here is some helpful information when setting up DeepSpeed ZeRO Stage 3 with Lig
 * Treat your GPU/CPU memory as one large pool. In some cases, you may not want to offload certain things (like activations) to provide even more space to offload model parameters
 * When offloading to the CPU, make sure to bump up the batch size as GPU memory will be freed
 * We also support sharded checkpointing. By passing ``save_full_weights=False`` to the ``DeepSpeedStrategy``, we'll save shards of the model which allows you to save extremely large models. However to load the model and run test/validation/predict you must use the Trainer object.
+* DeepSpeed provides `MiCS support <https://deepspeed.readthedocs.io/en/latest/zero3.html#deepspeed.runtime.zero.config.DeepSpeedZeroConfig.mics_shard_size>`_ which allows you to control how model parameters are sharded across GPUs. This can be useful if you have a large cluster of GPUs and want to avoid communication overhead.
```

Comment:
I would be great if you could add a simple example of MiCS configuration here, so people know how to pass it to the strategy, same for Fabric.

Reply:
Hi @hehepig4 , saw you mentioned PR #20474 , I think that PR might be better off to be scoped to only dealing with the `timeout` parameter, so I deleted the commit where I added `kwargs` to `DeepSpeedStrategy`. If you do feel the need to add them back, feel free to do it in your PR!